lib/node/pl-config/src/common/storages.ts

```diff
@@ -106,7 +113,7 @@ export async function createDefaultLocalStorages(workdir: string): Promise<Stora
   return storages;
 }
 
-export function newDefaultLocalStorages(workdir: string): StoragesSettings {
+function newDefaultLocalStorages(workdir: string, externalURL: string): StoragesSettings {
```

Comment:
Here we could just require externalURL as `string`, not as `?: string`, right? And then we won't need the runtime check below. Also, in JS world it's `externalUrl`, we don't capitalize abbreviations here (strange world 🤷🏼‍♂️)